Robobutler

kohsuke opened the pull request:
Adding robobutler to edamame

kohsuke committed:
Added manifest to install and run robobutler

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #48](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/48/) SUCCESS
This pull request looks good

rtyler commented:
I don't think we should merge this until this code manages the meeting output directory too :thumbsdown:

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #66](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/66/) FAILURE
Looks like there's a problem with this pull request

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #68](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/68/) FAILURE
Looks like there's a problem with this pull request

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #71](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/71/) FAILURE
Looks like there's a problem with this pull request

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #89](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/89/) FAILURE
Looks like there's a problem with this pull request

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #100](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/100/) FAILURE
Looks like there's a problem with this pull request

kohsuke commented:
As far as I'm concerned, this branch is ready.

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #102](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/102/) SUCCESS
This pull request looks good

Commit:
Fix the `lint` rake task to use the right `ignore_paths`

Otherwise, a local `r10k puppetfile install` would cause boatloads of lint
errors since the puppetlabs_spec_helper lint task overwrites ignore_paths
instead of adding to it
(https://github.com/jenkins-infra/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L162)

Commit:
Upgrade the Vagrantfile to use a Docker-capable 12.04 LTS AMI

Commit:
Add vagrant/test-environment specific hiera data

Commit:
Refactor the main docker management into the docker profile

This should be included by all profiles that need to use docker

Commit:
update butlerbot profile and add more test coverage

Commit:
Rework the butlerbot profile to use the puppetlabs/apache modules apache::vhost type

Commit:
Remove the jenkins_apache module from the tree, this exists in puppet-apache-logcompressor

Commit:
Add the puppet-apache-logcompressor module

This commit also reorganizes the .fixtures.yml file a bit to include more
modules that can be found directly on forge. It's a bit faster than a full git
clone

Commit:
Incorporate the log-compressor module into the robobutler profile

cloudbees-pull-request-builder commented:
[infra » jenkins-infra #110](https://jenkins.ci.cloudbees.com/job/infra/job/jenkins-infra/110/) SUCCESS
This pull request looks good

rtyler commented:
I've reworked this, and done a full test run on a Vagrant machine, I'm going to go ahead and merge to staging and give it a whirl!
